aggregate/aggregate

```diff
@@ -188,6 +198,91 @@ def main():
         "Other_resources"
     ]]
 
+    # Parse pdbminer complexes output:
+    for index, row in pdbminer_c_df.iterrows():
+        structure_id = row["structure_id"]
+        complex_details = row["complex_details"]
+
+        # Remove asterisks from structure IDs:
+        normalized_structures = final_df["Structure"].apply(lambda x: ";".join(s.replace("*", "").replace("**", "") for s in x.split(";")))
+
+        # Check if any structure_id is not in final_df:
+        if not any(structure_id in existing_structure.split(";") for existing_structure in normalized_structures):
```

Comment:
can we do the opposite?

here you are checking if this happens, then continue; can we do instead, if something this doesn't happen, pass to next iteration of the for loop? e.g.

```
if any(...):
     continue
```

in this way we avoid one extra indentation level and the cognitive load of having to remember this